src/main/java/org/jabref/gui/LibraryTab.java

```diff
@@ -118,6 +118,16 @@ public class LibraryTab extends Tab {
     private enum PanelMode { MAIN_TABLE, MAIN_TABLE_AND_ENTRY_EDITOR }
 
     private static final Logger LOGGER = LoggerFactory.getLogger(LibraryTab.class);
+    // Mapping of Git status indicators, as a static constant of the class
+    private static final Map<GitHandler.GitStatus, String> GIT_STATUS_INDICATORS = Map.of(
+        GitHandler.GitStatus.MODIFIED, "[Modified]",
+        GitHandler.GitStatus.STAGED, "[Staged]",
+        GitHandler.GitStatus.AHEAD_OF_REMOTE, "[Ahead]",
+        GitHandler.GitStatus.BEHIND_REMOTE, "[Behind]",
+        GitHandler.GitStatus.UP_TO_DATE, "[Up to date]",
+        GitHandler.GitStatus.COMMITTED, "[Committed]",
+        GitHandler.GitStatus.UNTRACKED, "[Untracked]"
+    );
```

Comment:
Data-oriented design would suggest to make a switch-expression out of this lookup-table, that lacks a default statement, so the compiler would complain if the programmer forgot to implement a missing status.